packages/cli/src/commands/help.ts

```diff
@@ -124,7 +125,11 @@ export function buildCommandSynopsisLine(command: Command, parent?: Command) {
 
   if (args.length > 0) {
     for (const argument of args) {
-      line.push(argument.required ? argument.name : `[${argument.name}]`);
+      let { name } = argument;
+      if (argument.multiple) {
+        name += '...';
```

Comment:
Can we add a space between the argument name and the `...`? 

Adding the space would be consistent with how unix shows multiple args in `man` pages:
```
rm [-f | -i] [-dIRrvWx] file ...
```